### PR TITLE
maint(ci): align coverage reporting with Pro for WSL project

### PR DESCRIPTION
**Pull request by didrocks**
We use a directory strategy to ship it, which is compatible with multiple platforms using a relative directory, and rely on autosearch so that only this subdirectory contains all codecov related content.

### CODE REVIEW

**Review by denisonbarbosa — APPROVED** [review]
Looks good to me! It's definitely better to have our projects in synchrony with each other as much as possible.

**Comment by didrocks** [comment]
Forgot to mark it as ready (was waiting on the WSL side to be merged first). Already got the approval from Denis, rebasing and merging.

**Pull request merged**
